docs/getting-started-apm-server.asciidoc

```diff
@@ -0,0 +1,317 @@
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-linux-x86_64.tar.gz
+tar xzvf apm-server-{version}-linux-x86_64.tar.gz
+------------------------------------------------
+endif::[]
+
+[[mac]]
+*Mac:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-darwin-x86_64.tar.gz
+tar xzvf apm-server-{version}-darwin-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[[installing-on-windows]]
+*Windows:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+. Download the APM Server Windows zip file from the
+https://www.elastic.co/downloads/apm/apm-server[downloads page].
+
+. Extract the contents of the zip file into `C:\Program Files`.
+
+. Rename the `apm-server-<version>-windows` directory to `APM-Server`.
+
+. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select *Run As Administrator*).
+If you are running Windows XP, you may need to download and install PowerShell.
+
+. From the PowerShell prompt, run the following commands to install APM Server as a Windows service:
++
+[source,shell]
+----------------------------------------------------------------------
+PS > cd 'C:\Program Files\APM-Server'
+PS C:\Program Files\APM-Server> .\install-service-apm-server.ps1
+----------------------------------------------------------------------
+
+NOTE: If script execution is disabled on your system,
+you need to set the execution policy for the current session to allow the script to run.
+For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-apm-server.ps1`.
+
+endif::[]
+
+[[docker]]
+*Docker:*
+
+See <<running-on-docker, Running on Docker>> for deploying Docker containers.
+
+// *******************************************************
+// STEP 2
+// *******************************************************
+
+[[apm-server-configuration]]
+=== Step 2: Set up and configure
+
+[float]
+==== Elastic Cloud
+
+If you're running APM Server in Elastic cloud, you can configure your own user settings right in the Elasticsearch Service Console.
+Any changes are automatically appended to the `apm-server.yml` configuration file for your instance.
+
+Full details are available in the {cloud}/ec-manage-apm-settings.html[APM user settings] documentation.
+
+[float]
+==== Self installation
+
+Once you've installed {beatname_uc}, you can set up and configure it according to your needs.
+
+[float]
+[[index-template-config]]
+===== Set up
+
```

Comment:
While the level you use is the right one, the visual representation does not clearly show that _Set up_ is nested under _Self installation_.  I am wondering if this additional _Setup_ and _Configure_ headers are really needed or you could just remove them and having everything directly under _Self installation_. 

<img width="736" alt="Screenshot 2019-09-11 at 11 40 31" src="https://user-images.githubusercontent.com/5555349/64686629-3e291a80-d489-11e9-9579-8729131a55d3.png">